spec/plainmonthday.html

```diff
@@ -348,6 +349,10 @@ <h1>ToTemporalMonthDay ( _item_ [ , _options_ ] )</h1>
       <emu-alg>
         1. If _options_ is not present, set _options_ to *undefined*.
         1. Assert: Type(_options_) is Object or Undefined.
+        1. If _options_ is not *undefined*, then
+          1. Let _resolvedOptions_ be OrdinaryObjectCreate(*null*).
```

Comment:
These 2 lines seem repeated a bunch; is it worth an AO for them?

Reply:
Addressed in #2562, there were several instances of this pattern already.